CMakeLists.txt

```diff
@@ -25,17 +39,26 @@ else()
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -funsigned-char -Wall -Wno-pedantic -Wno-sign-compare")
 endif()
 
-file(GLOB SOURCES SRC/*.H SRC/*.CPP SRC/EFP/*.H SRC/EFP/*.CPP SRC/ERROR/*.H SRC/ERROR/*.CPP SRC/NET/*.H SRC/NET/*.CPP)
+file(GLOB SOURCES SRC/*.H SRC/*.CPP SRC/EFP/*.H SRC/EFP/*.CPP SRC/ERROR/*.H SRC/ERROR/*.CPP SRC/NET/*.H SRC/NET/*.CPP CONFIG.H)
 
 if (WIN32)
 	file(GLOB MSVC_SOURCES SRC/MSVC/*.H SRC/MSVC/*.CPP)
 	list(APPEND SOURCES ${MSVC_SOURCES})
 endif()
 
-add_executable(tk3 ${SOURCES})
-target_include_directories(tk3 SYSTEM PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/SRC ${SDL2_INCLUDE_DIR} ${SDL2_MIXER_INCLUDE_DIRS} ${SDL2_NET_INCLUDE_DIRS})
-target_link_libraries(tk3 ${SDL2_LIBRARY} ${SDL2_MIXER_LIBRARIES} ${SDL2_NET_LIBRARIES})
+add_executable(ultimatetapankaikki ${SOURCES})
+target_include_directories(ultimatetapankaikki SYSTEM PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/SRC ${SDL2_INCLUDE_DIR} ${SDL2_MIXER_INCLUDE_DIRS} ${SDL2_NET_INCLUDE_DIRS})
+target_link_libraries(ultimatetapankaikki ${SDL2_LIBRARY} ${SDL2_MIXER_LIBRARIES} ${SDL2_NET_LIBRARIES})
 
 if (WIN32)
-	set_target_properties(tk3 PROPERTIES OUTPUT_NAME "GAME" SUFFIX ".EXE")
+	set_target_properties(ultimatetapankaikki PROPERTIES OUTPUT_NAME "GAME" SUFFIX ".EXE")
 endif()
+
+install(TARGETS ultimatetapankaikki DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(FILES OPTIONS.CFG DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
+install(DIRECTORY EFPS DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
```

Comment:
`install(DIRECTORY EFPS FNTS LEVS MUSIC WAVS DESTINATION ...` should work according to documentation.

Reply:
Fixed